temel_bilgiler/html_css/html-temelleri/html-boilerplate.md

```diff
@@ -0,0 +1,183 @@
+
+Bu daha sonra JavaScript kullanarak HTML'yi manipüle etmeyi öğrendiğimizde daha da önem kazanır. Şimdilik sadece HTML öğesinin her HTML belgesinde yer alması gerektiğini bilin.
+
+`index.html` dosyasına geri dönelim ve `<html>` öğesini açılış ve kapanış etiketleriyle ekleyelim, şu şekilde:
+
+~~~html
+<!DOCTYPE html>
+<html lang="en">
+</html>
+~~~
+
+#### `lang` özelliği nedir?
+
+`lang`, o öğenin metin içeriğinin dilini belirtir. Bu özellik web sayfasının erişilebilirliğini geliştirmek için kullanılır. Yardımcı teknolojilerin; örneğin ekran okuyucuların, dilin doğru bir şekilde telaffuz edilmesi için dilinize göre uyarlanmasına olanak tanır.
+
+### Head öğesi
+
+`<head>` öğesi web sayfalarımız **hakkında** önemli meta bilgilerini ve web sayfalarımızın tarayıcıda doğru bir şekilde görüntülenmesi için gereken şeyleri yerleştirdiğimiz yerdir.
+`<head>` içinde, web sayfasında içerik gösteren herhangi bir öğe kullanmamalıyız.
+
+#### Karakter kümesi meta öğesi
+
+Web sayfasının karakter kümesi kodlaması için her zaman head öğesinde `<meta charset="utf-8">` meta etiketine sahip olmalıyız.
+
+Kodlamayı ayarlamak çok önemlidir çünkü bu web sayfasının tarayıcıda farklı dillerden gelen özel sembollerin ve karakterlerin doğru bir şekilde görüntülenmesini sağlar.
+
+#### Başlık öğesi
+
+HTML belgesinin baş kısmında her zaman yer alması gereken başka bir öğe, `title` öğesidir:
+
+ `<title>My First Webpage</title>`
+
+`title` öğesi, web sayfalarına insanlar tarafından okunabilir bir başlık vermek için kullanılır ve bu başlık web sayfamızın tarayıcı sekmesinde görüntülenir.
+
+Eğer bir `title` öğesi eklemeseydik, web sayfasının başlığı varsayılan olarak dosya adı olurdu. Bizim durumumuzda bu `index.html` olurdu ve kullanıcılar için çok anlamlı olmayan bir başlık olurdu; kullanıcının birçok tarayıcı sekmesi açık ise web sayfamızı bulmak oldukça zorlaşırdı.
+
+HTML belgesinin `head` kısmına yerleştirilebilecek birçok başka öğe bulunmaktadır. Ancak şu anda burada ele aldığımız iki öğeyi bilmek önemlidir. Geri kalan eğitim sürecinde baş kısmına yerleştirilecek daha fazla öğeyi tanıtacağız.
+
+`index.html` dosyamıza geri dönelim ve içinde karakter kümesi meta öğesi ve bir başlık içeren bir `head` öğesi ekleyelim. `head` öğesi, HTML öğesinin içinde yer almalı ve açılış `<html>` etiketi altındaki ilk öğe olmalıdır.
+
+~~~html
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>My First Webpage</title>
+  </head>
+</html>
+~~~
+
+### Body öğesi
+
+HTML şablonunu tamamlamak için gereken son öğe `<body>` öğesidir. Bu, kullanıcılara görüntülenecek tüm içeriğin (metin, resimler, listeler, bağlantılar vb) yer alacağı yerdir. 
+
+Boilerplate'i tamamlamak için `index.html` dosyasına bir body öğesi ekleyin. Body öğesi de HTML öğesinin içine yerleştirilir ve her zaman `head` öğesinin altında yer alır, aşağıdaki gibi:
```

Comment:
```suggestion
Şablonu tamamlamak için `index.html` dosyasına bir `body` öğesi ekleyin. Body öğesi de HTML öğesinin içine yerleştirilir ve her zaman head öğesinin altında yer alır, aşağıdaki gibi:
```

Reply:
boilerplate'i çok fazla türkçeleştirmek istedim ancak anlamı oturmadığı için öyle bırakma kararı aldım. bir kaç yazılımcının bu terimin türkçeleştirilmesi hakkında tweetlerini de inceleyince hepsiyle aynı fikirde olup direkt ana haliyle kalması kararı almıştım. 

bundan dolayı dosyanın adını da html-boilerplate.md olarak bıraktım.

Reply:
Selam @gorkemdolcek, vermis oldugun katkidan dolayi cok tesekkur ederiz!

Evet, `boilerplate` sektorde cok fazla yer edinmis bir kelime olsa da, Turkce'sini okumaya gelen birinin `boilerplate` kelimesini bilmesini bekleyemeyiz diye dusunuyorum. Bu yuzden `şablon` kelimesiyle degistirmeyi uygun gordum.

Reply:
Tekrar geri donuyorum ve boilerplate olarak kalmasinin daha iyi oldugunu dusundum ben de. Boylelikle sozluk.kamp.us'ten edinmek istedigimiz verimi de bu sekilde saglayabiliriz. 

Sozluk => Dijital Terimler Sozlugu => Bu tarz kelimelerin anlamlarini orada kaydediyor olucaz, o sayede bu kelimenin ustune gelindiginde sozlukle alakali bisiler yapabiliriz